### ISSUE TITLE
once you have an error, Haskelly stop working (even if you fix it and save again)

### ISSUE THREAD

**Issue by rvion** [haskelly-dev/Haskelly#24]
Intero fails to start silently

no messages nor feedback in the UI:

![image](https://cloud.githubusercontent.com/assets/2150990/22968916/b1382134-f36c-11e6-8c9a-4138a48acfb7.png)

if I open Developper tools, I see 

![image](https://cloud.githubusercontent.com/assets/2150990/22968933/c75bf9a4-f36c-11e6-8b55-5a813004212b.png)

intero is running fine in my terminal
```shell
*Lib> :all-types
src/Lib.hs:(6,1)-(6,9): IO ()
src/Lib.hs:(7,5)-(7,13): String -> IO ()
src/Lib.hs:(7,14)-(7,30): [Char]
src/Lib.hs:(7,5)-(7,30): IO ()
src/Lib.hs:(6,12)-(7,30): IO ()
```

everything seems up to date:

```shell 
stack install stack-run intero QuickCheck
Copying from /Users/rvion/.stack/snapshots/x86_64-osx/lts-8.0/8.0.2/bin/intero to /Users/rvion/.local/bin/intero
Copying from /Users/rvion/dev/hsss/.stack-work/install/x86_64-osx/lts-8.0/8.0.2/bin/stack-run to /Users/rvion/.local/bin/stack-run
Copied executables to /Users/rvion/.local/bin:
- intero
- stack-run

ll /usr/local/bin/intero
lrwxr-xr-x  1 root  admin    30B Feb 15 10:49 /usr/local/bin/intero -> /Users/rvion/.local/bin/intero

uname -a
Darwin MacBook-Pro-de-administrateur.local 15.6.0 Darwin Kernel Version 15.6.0: Thu Jun 23 18:25:34 PDT 2016; root:xnu-3248.60.10~1/RELEASE_X86_64 x86_64

code --version
1.9.1
f9d0c687ff2ea7aabd85fb9a43129117c0ecf519

stack --version
Version 1.2.1, Git revision f28002bdea51468566fd9370f7f9e801c2b51bc5 x86_64 hpack-0.14.1
```

**Comment by martrik** [comment]
Hi! From what I can see I think that the issue is that your file is not compiling. Until there's not a first version which correctly loads into GHCi, Haskelly is not able to provide code completion or type hovers. However, it is true that we should provide a better message than 'Loading...'. We'll include that in the next release!

**Comment by rvion** [comment]
I think the problem is more important than that, see my other issue here: https://github.com/haskelly-dev/Haskelly/issues/29

**Comment by rvion** [comment]
![image](https://cloud.githubusercontent.com/assets/2150990/22973565/76cccf38-f37e-11e6-8049-a0b9eec4847f.png)

**Comment by martrik** [comment]
This error was caused by an unfortunate regression, sorry for any trouble caused. It has been fixed in 0.4.15

**Comment by rvion** [comment]
thanks !
